lib/attr_encrypted.rb

```diff
@@ -277,6 +310,7 @@ module InstanceMethods
     #  @user = User.new('some-secret-key')
     #  @user.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
     def decrypt(attribute, encrypted_value)
+      self.class.encrypted_attributes[attribute.to_sym][:operation] = :decrypting
```

Comment:
@shuber any thoughts regarding this new commit? https://github.com/attr-encrypted/attr_encrypted/commit/e96f13e868189a8592f1159108113f3e008d850f